star-rx72n-firmware/Dockerfile

```
@@ -1,14 +1,158 @@
+      --disable-libgomp \
+      --disable-libatomic \
+      --disable-libsanitizer \
+      --disable-libstdcxx && \
+    make -j"${MAKE_JOBS}" && \
+    make install
+
+
+FROM ubuntu:22.04
 
 # Install build dependencies (cached in image)
 RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)